### PR TITLE
Code for test case DA_DP_TC072

### PR DESCRIPTION
**Pull request by lqthai07t3**
- Code for test case DA_DP_TC072.
- Run code and check.

### CODE REVIEW

**Comment by lqthai07t3** [comment]
image

File: TADashboard/src/test/java/tests/general_settings/PositiveDPGeneralSettingsTests.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        itemTypes.add("Data Sets");
+        itemTypes.add("Actions");
+        itemTypes.add("Interface Entities");
+        itemTypes.add("Test Results");
+        itemTypes.add("Test Case Results");
+        itemTypes.add("Test Suites");
+        itemTypes.add("Bugs");
+
+        Logger.verify("Verify that all data profile types are listed under 'Item Type' dropped down menu");
+        for (String itemType : itemTypes) {
```
Comment (tanmle) [inline TADashboard/src/test/java/tests/general_settings/PositiveDPGeneralSettingsTests.java]: why don't you just assert the list?

**Review by tanmle — CHANGES_REQUESTED** [review]

File: TADashboard/src/test/java/tests/general_settings/PositiveDPGeneralSettingsTests.java (hunk 1 above)
Comment (lqthai07t3) [reply TADashboard/src/test/java/tests/general_settings/PositiveDPGeneralSettingsTests.java]: Okie, It's a good idea.

Comment (lqthai07t3) [reply TADashboard/src/test/java/tests/general_settings/PositiveDPGeneralSettingsTests.java]: I have changed it. Please, check again

**Comment by lqthai07t3** [comment]
image

File: TADashboard/src/main/java/page_objects/DataProfilesGeneralSettingsPage.java [hunk 2]
```diff
@@ -25,4 +27,8 @@ public void createNewProfile(String name, String type, String relatedData) {
         cbbRelatedData.select(relatedData);
         btnFinish.click();
     }
+
+    public boolean areItemTypesDisplayedInOptions(List<String> listItems) {
```
Comment (tanmle) [inline TADashboard/src/main/java/page_objects/DataProfilesGeneralSettingsPage.java]: you should get the options list then compare in the test script.

**Review by tanmle — CHANGES_REQUESTED** [review]

File: TADashboard/src/main/java/page_objects/DataProfilesGeneralSettingsPage.java (hunk 2 above)
Comment (lqthai07t3) [reply TADashboard/src/main/java/page_objects/DataProfilesGeneralSettingsPage.java]: Yeah, Okie. I got it

Comment (lqthai07t3) [reply TADashboard/src/main/java/page_objects/DataProfilesGeneralSettingsPage.java]: I have updated code. Please check again.

image

**Review by MinhQuan119 — APPROVED** [review]

File: TADashboard/src/main/java/page_objects/DataProfilesGeneralSettingsPage.java [hunk 3]
```diff
@@ -25,4 +27,8 @@ public void createNewProfile(String name, String type, String relatedData) {
         cbbRelatedData.select(relatedData);
         btnFinish.click();
     }
+
+    public List<String> getAllItemTypesOptions() {
```
Comment (tanmle) [inline TADashboard/src/main/java/page_objects/DataProfilesGeneralSettingsPage.java]: get all item types is fine.

File: TADashboard/src/test/java/tests/general_settings/PositiveDPGeneralSettingsTests.java [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+        listItemTypes.add("Data Sets");
+        listItemTypes.add("Actions");
+        listItemTypes.add("Interface Entities");
+        listItemTypes.add("Test Results");
+        listItemTypes.add("Test Case Results");
+        listItemTypes.add("Test Suites");
+        listItemTypes.add("Bugs");
+
+        Logger.verify("Verify that all data profile types are listed under 'Item Type' dropped down menu");
+        Assert.assertTrue(dataProfilesGeneralSettingsPage.getAllItemTypesOptions().containsAll(listItemTypes),
```
Comment (tanmle) [inline TADashboard/src/test/java/tests/general_settings/PositiveDPGeneralSettingsTests.java]: what is the difference between containsAll and asserting equals 2 Lists?

**Review by tanmle — CHANGES_REQUESTED** [review]

File: TADashboard/src/main/java/page_objects/DataProfilesGeneralSettingsPage.java (hunk 3 above)
Comment (lqthai07t3) [reply TADashboard/src/main/java/page_objects/DataProfilesGeneralSettingsPage.java]: Yeah, I will fix it for next commit. Thank you so much

File: TADashboard/src/test/java/tests/general_settings/PositiveDPGeneralSettingsTests.java (hunk 4 above)
Comment (lqthai07t3) [reply TADashboard/src/test/java/tests/general_settings/PositiveDPGeneralSettingsTests.java]: Let's take a look for your your question. Thank you

**Review by tanmle — APPROVED** [review]

**Pull request merged**